pkg/resources/resources.go

```diff
@@ -431,6 +434,10 @@ const (
 	// configuration.
 	MachineControllerMutatingWebhookConfigurationName = "machine-controller.kubermatic.io"
 
+	// MachineValidatingWebhookConfigurationName is the name of the machine validating webhook
+	// configuration.
+	MachineValidatingWebhookConfigurationName = "machine.kubermatic.io"
+
```

Comment:
We don't need these additions

Reply:
right, i split this PR from my code with a new Machine webhook and this was leftover, will remove